.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+AllCops:
```

Comment:
We should skip this and just use the default so we cleanup all the files

Reply:
Removed the rubocop config and ran linter on the codebase in https://github.com/chef/corefoundation/pull/10/commits/1e500d2555efe0d2fe85b2a0dc27e22fdbc0b63d 👍🏼